app/javascript/components/pages/ApartmentShow.test.js

```diff
@@ -1,30 +1,25 @@
-import React from "react"
-import { render, screen } from "@testing-library/react"
-import ApartmentShow from "./ApartmentShow"
-import { MemoryRouter, NavLink, Routes, Route } from "react-router-dom"
-
-const renderComponent = (Apartments) => {
+import React from "react";
+import { render, screen } from "@testing-library/react";
+import ApartmentShow from "./ApartmentShow";
+import { MemoryRouter, NavLink, Routes, Route, BrowserRouter } from "react-router-dom";
+import mockApartments from "../mockApartments"
+import userEvent from "@testing-library/user-event";
 
+const thisApartment = () => {
   render(
     <MemoryRouter initialEntries={["/apartmentshow/1"]}>
       <Routes>
-          <Route path="apartmentshow/:id" element={<ApartmentShow apartments={Apartments} />}/>
+        <Route
+          path="apartmentshow/:id"
+          element={<ApartmentShow apartments={mockApartments} />}
+        />
       </Routes>
     </MemoryRouter>
-  )
-}
+  );
+};
 
 describe("<ApartmentShow />", () => {
-
-  it("renders a card for each Apartment", (Apartments) => {
-
-    render(
-      <ApartmentShow apartments={Apartments} />
-    );
-
-    Apartments.map((apartment) => {
-      const nameElement = screen.getByText(apartment.name);
-      expect(nameElement).toBeInTheDocument();
-    });
+  it("has functioning navigation links", () => {
```

Comment:
Can we add some tests in here to check the nav links and/ or what is appearing on the page?